src/benchmark_mediation.py

```diff
@@ -1197,11 +1195,9 @@ def med_dml(
         w_t1_x,  # E[E[Y|T=0,M,X]|T=1,X]
         mu_t1_x,  # E[Y|T=1,X]
         mu_t0_x,  # E[Y|T=0,X]
-    ) = [np.empty((max(crossfit, 1),), dtype=object) for _ in range(12)]
+    ) = [np.zeros(n) for _ in range(10)]
```

Comment:
what does this `10` mean ? Is it a number of iterations ? Please give it a name, and set it to 10.

Reply:
It is the number of nuisance parameters we initialize to zero. Okay I'll name it, but I don't think it is a good idea because this number is not supposed to change and doesn't really have any meaning

Notice the same thing is done in all others estimators (AIPW, IPW, g-computation, multiply-robust), so we should declare an issue if we change it

Reply:
I had not see the context. You can ignore this comment